src/paperless_tika/parsers.py

```diff
@@ -62,7 +66,9 @@ def parse(self, document_path: Path, mime_type: str, file_name=None):
         if self.text is not None:
             self.text = self.text.strip()
 
-        self.date = parsed.created
+        tz = timezone.get_current_timezone()
+
+        self.date = timezone.make_aware(parsed.created, tz)
```

Comment:
This change is useful, with a few tweaks.

- The create date might be `None`
- The create date might already be aware
- There's no need to get the current timezone, Django will use the configured one